### PR TITLE
feat(oci) remove dead code pkg/art

### PR DESCRIPTION
**Pull request by bitsf**
Signed-off-by: Ziming Zhang <zziming@vmware.com>

### CODE REVIEW

File: src/pkg/artifact/candidate.go [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package art
+package artifact
```
Comment (ywk253100) [inline src/pkg/artifact/candidate.go]: According to the "single responsibility principle" for pkg, please move the selector related code into a separated package "src/pkg/selector"

**Comment by ywk253100** [comment]
@bitsf Please resolve the conflicts

**Comment by renmaosheng** [comment]
just curios, why the coverage is dropping?

**Comment by bitsf** [comment]
> just curios, why the coverage is dropping?
image

It seems some UI case dropped, but I also don't know why yet, this PR did nothing with UI code, maybe the calculate comparison commit is not correct

**Pull request merged**